docs/source/application_development/getting_started.rst

```diff
@@ -159,7 +159,7 @@ with ethereum accounts, so be sure to specify an account you control with a ``Si
    True
 
    # The root directory containing the private keys
-   >>> keyring.keyring_root
+   >>> keyring.keystore_dir
```

Comment:
Can't comment on the code above directly, but should the line 146
```
from nucypher.config.keyring import NucypherKeyring
```
be
```
from nucypher.crypto.keystore import Keystore
```
?

Reply:
Ah yes, you're right.  I forgot to review the documentation affected by this PR.  I'll make an update.

Reply:
✔️